docs/project/about.md

```diff
@@ -6,13 +6,27 @@ Pyodide brings the Python runtime to the browser via WebAssembly, along with
 the Python scientific stack including NumPy, Pandas, Matplotlib, parts of
 SciPy, and NetworkX. The [packages
 directory](https://github.com/iodide-project/pyodide/tree/master/packages)
-lists over 35 packages which are currently available.
+lists over 75 packages which are currently available. In addition it's possible
+to install pure Python wheels from PyPi.
 
 Pyodide provides transparent conversion of objects between Javascript and
 Python. When used inside a browser, Python has full access to the Web APIs.
 
-While closely related to the [iodide project](https://iodide.io), a tool for
-*literate scientific computing and communication for the web*, Pyodide goes
-beyond running in a notebook environment. To maximize the flexibility of the
-modern web, **Pyodide** may be used standalone in any context where you want to
-**run Python inside a web browser**.
+## History
+
+Pyodide was created in 2018 by [Michael Droettbom](https://github.com/mdboom)
+at Mozilla as part of the [iodide project](https://iodide.io), a set of experiments around
+scientific computing and communication for the web.
+
+At present Pyodide is an independent and community driven open-source project.
+The decision making process is outlined in the {ref}`project governance <project-governance>`.
+Pyodide may be used standalone in any context where you want to run Python
+inside a web browser.
+
+## Infrastructure support
+
+We would also like to thank,
+ - [Mozilla](https://www.mozilla.org/en-US/) and
+[CircleCl](https://circleci.com/) for Continuous Integration resources
```

Comment:
```suggestion
- [CircleCl](https://circleci.com/) for Continuous Integration resources
```

Reply:
That was meant to be a single bullet point, but we can certainly split it if there are more thing to put for Mozilla support :)

More seriously, we can split if you have suggestions what to put there because I think,
```
 - Mozilla and
 - CircleCI for Continuous Integration resources
```
looks a bit strange.

Reply:
oh sorry, i was quickly scanning over it and didn't see the and at the end of the line. you can discard this :)